src/modules/removeTask.js

```diff
@@ -0,0 +1,16 @@
+import getLocalStorage from './getStorage.js';
+import setLocalStorage from './setStorage.js';
+import updateArray from './updateArray.js';
+
+const removeTask = (id) => {
+  let taskArray = getLocalStorage();
+  if (id >= 0) {
+    taskArray = taskArray.filter((task) => task.id !== id);
+  } else {
+    taskArray = taskArray.filter((task) => !task.completed);
```

Comment:
- [ ] Checking a task as completed does not change the corresponding task value of the local storage. Kindly ensure that checking a task as completed changes the value of that task from `false` to `true`, as informed in the previous review